src/components/randomize.jsx

```diff
@@ -44,8 +44,15 @@ export default class Randomize extends React.Component {
   }
 
   render() {
-    return (
-      <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>      
-    )
+    if(this.props.enabled == "enabled"){
+      return (
+        <button className="button" className="btn btn-outline-danger btn-sm" onClick={this.handleClick}>Randomize Colors</button>
+      )
+    }
+    else{
+      return (
+        <button className="button" className="btn btn-outline-danger btn-sm" disabled={ this.props.enabled } onClick={this.handleClick}>Randomize Colors</button>
```

Comment:
Sean did randomize but you should go ahead and remove it since we caught the bug now.
It doesn't make sense to have two separate attributes for className when it can either be squashed into one, or removed -- especially because className="button" does nothing because it's not even in Bootstrap 4.

Or change className="button" to type="button".

I got the alert graphic, but it's confusing still...in that case could you just remove the -outline attribute from the outline buttons? that way there won't be a hover color.